README.md

```diff
@@ -32,26 +34,20 @@ Core Server (i.e. SERVER) project are **public**.
 ## Installation
 `MongoSwift` works with Swift 4.0+.
 
-### FIRST: Install the MongoDB C Driver
-Because the driver wraps the MongoDB C driver, using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. The minimum required version of the C Driver is **1.13.0**.
+### OS X and Linux
 
-On a Mac, you can install both components at once using [Homebrew](https://brew.sh/):
-`brew install mongo-c-driver`.
-
-Or on Linux, use `apt-get` to install `libmongoc` (which includes `libbson` as a dependency) and `pkg-config` (which enables Swift Package Manager to find the components):
-```
-sudo apt-get install pkg-config
-sudo apt-get install libmongoc-1.0.0
-```
+Installation on OS X and Linux is supported via [Swift Package Manager](https://swift.org/package-manager/).
 
-Alternatively, see the [installation guide](http://mongoc.org/libmongoc/current/installing.html) from libmongoc's documentation.
+#### Step 1: Install the MongoDB C Driver
+The driver wraps the MongoDB C driver, and using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. **The minimum required version of the C Driver is 1.13.0**.
 
-Next, see instructions for installation with either Swift Package Manager or CocoaPods in the following sections.
+On a Mac, you can install both components at once using [Homebrew](https://brew.sh/):
+`brew install mongo-c-driver`.
 
-### NEXT: Install the Driver Using Swift Package Manager
-*Please make sure you have followed the instructions in the previous section on installing the MongoDB C Driver before proceeding.*
+On Linux: please follow the [instructions](http://mongoc.org/libmongoc/current/installing.html#building-on-unix) from `libmongoc`'s documentation. Note that the versions provided by `apt-get` may be too old, in which case you can follow the instructions for building and installing from source.
```

Comment:
I would just suggest replacing `apt-get` with "your package manager" to keep this distribution agnostic. I dunno if we make other assumptions about Linux OS elsewhere in this README. LGTM otherwise.

Reply:
done. I think this section is the only part where linux is discussed.